snap/hooks/install

```diff
@@ -1,13 +1,7 @@
 #!/bin/bash
 set -x
 
-if [ "$(snapctl get daemon)" = "" ]
-then
-  if grep -q -e snap_core= -e snapd_recovery_mode= /proc/cmdline
-  then snapctl set daemon=true
-  else snapctl set daemon=false
-  fi
-fi
+${SNAP}/include/hook-install
 
 source "${SNAP}/include/hook-common"
```

Comment:
It looks odd to execute one of the scripts in `${SNAP}/include/` and source the other

Reply:
Moved to `bin/`.